crates/rpc/rpc-api/src/address.rs

```diff
@@ -0,0 +1,17 @@
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use reth_primitives::{Address, BlockNumber, TxIndex};
+
+/// address_ namespace RPC interface.
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "address"))]
+#[cfg_attr(feature = "client", rpc(server, client, namespace = "address"))]
+pub trait AddressApi {
+    /// Returns transaction identifiers relevant to an address.
+    ///
+    /// See: https://github.com/ethereum/execution-apis/pull/453/
+    #[method(name = "getAppearances")]
+    async fn get_appearances(
+        &self,
+        address: Address,
+        range: Option<(BlockNumber, BlockNumber)>,
+    ) -> RpcResult<Vec<(BlockNumber, TxIndex)>>;
```

Comment:
there's no spec for this yet, so we should think about some examples here and how this is parsed.

should the range be an object?
and the response a struct with named fields?

Reply:
Good suggestion. I have changed the range to be an object. 

The return type is also now spec compliant (lives in ethereum/execution-apis). Serde unit tests added.